db/migrations/versions/011_event_data_to_jsonb.py

```diff
@@ -0,0 +1,169 @@
+"""empty message
```

Comment:
there's a migration in feature production by Marc, so check the number/id on this before committing